blivet/threads.py

```diff
@@ -33,7 +34,7 @@
 
 def exclusive(m):
     """ Run a callable while holding the global lock. """
-    @wraps(m)
+    @wraps(m, set(functools.WRAPPER_ASSIGNMENTS) & set(dir(m)))
```

Comment:
The python2 version of wraps apparently doesn't automatically adjust the list of assignments for a method as opposed to a function. My understanding is that this change is a no-op on python3.